### PR TITLE
add link to download stats

### PR DESCRIPTION
**Pull request by gdams**
This PR adds a handy per release link to the new download dashboard at http://dash.adoptopenjdk.net/

##### Checklist

- [x] `npm test` passes
- [ ] documentation is changed or added
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/js/archive.js [hunk 1]
```diff
@@ -45,12 +45,14 @@ function buildArchiveHTML(releases, jckJSON) {
 
     // set values for this release, ready to inject into HTML
     var publishedAt = moment(eachRelease.timestamp);
+    console.log(eachRelease)
```
Comment (supahgreg) [inline src/js/archive.js]: Should this be removed?

Comment (gdams) [reply src/js/archive.js]: updated PTAL

**Comment by gdams** [comment]
> Maybe not a huge deal, but the dashboards aren't currently mobile-friendly.

@supahgreg the dashboard isn't hosted in this repo so not a blocker in landing this. I agree that improvements can be made but no need right now.

**Comment by supahgreg** [comment]
@gdams Not hosted in this repo, but this PR would make it generally accessible.  I don't think mobile support is a huge deal initially, but wanted to bring it up.

**Comment by tellison** [comment]
Looks rubbish on macOS Chrome Version 70.0.3538.77 (Official Build) (64-bit) too
screen shot 2018-11-09 at 09 47 28

**Comment by karianna** [comment]
> Looks rubbish on macOS Chrome Version 70.0.3538.77 (Official Build) (64-bit) too
> screen shot 2018-11-09 at 09 47 28

Yep see https://github.com/AdoptOpenJDK/openjdk-dashboard/issues/1

**Pull request merged**